ffi/diplomat/Cargo.toml

```diff
@@ -72,8 +72,8 @@ writeable = { version = "0.4", path = "../../utils/writeable/" }
 # The version here can either be a `version = ".."` spec or `git = "https://github.com/rust-diplomat/diplomat", rev = ".."`
 # Since this crate is published, Diplomat must be published preceding a new ICU4X release but may use git versions in between
 # ALSO MAKE SURE TO UPDATE ffi_coverage/Cargo.toml!
-diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "dd70de7573e1ef30b312de04bb7e9f64e67bd19a" }
-diplomat-runtime = { git = "https://github.com/rust-diplomat/diplomat", rev = "dd70de7573e1ef30b312de04bb7e9f64e67bd19a" }
+diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "82e2e74c1f3c04b0360529aa7c019b2c56f3216d" }
```

Comment:
`./ffi_coverage/Cargo.toml` also needs to be updated

Reply:
Done.